composer.json

```diff
@@ -19,7 +19,8 @@
         "doctrine/cache": "1.*",
         "doctrine/collections": "1.*",
         "doctrine/lexer": "1.*",
-        "doctrine/annotations": "1.*"
+        "doctrine/annotations": "1.*",
+        "doctrine/event-manager": "1.*@dev"
```

Comment:
`^1.0.0@dev` (without `@dev` once we release it)?

Reply:
I replaced all constraints with caret. 👍

Reply:
bumped :+1: